--- uniconvertor-1.1.5/setup.py.orig	2010-06-20 21:55:03.000000000 +0200
+++ uniconvertor-1.1.5/setup.py	2010-12-04 14:14:29.906121097 +0100
@@ -185,8 +185,8 @@
 			},
 			
 			package_data={'uniconvertor.app': ['VERSION'],		
-			'uniconvertor': ['GNU_GPL_v2', 'GNU_LGPL_v2', 'COPYRIGHTS',
-							'share/icc/*.*', 'share/fonts/*.*', 'share/ps_templates/*.*'], 			
+			'uniconvertor': [
+							'share/icc/*.*', 'share/ps_templates/*.*'],
 			'sk1.app.modules': ['descr.txt',]
 			},
 
